lib/publiccloud/ssh_interactive.pm

```diff
@@ -28,7 +28,7 @@ sub establish_tunnel_console {
     enter_cmd("clear");
     enter_cmd('ssh -t sut');
     # give the ssh connection some time to settle
-    sleep 5;
+    sleep 10;
```

Comment:
Not too familiar with this but wouldn't it be possible to check the connection few times in a loop instead of increasing the sleep time?

Reply:
Good question,  I think it might be possible but I would like to keep the `sleep $N` style. It is simple.

Reply:
Yes, I know this.
Let me try to find a way to avoid using sleep here.